### DISCUSSION TITLE
WebServer at ip 0.0.0.0 when Wifi is off

### DISCUSSION THREAD

**Discussion by cosmaz** [arendst/Tasmota#14185] [Q&A] [answered]
Please tell me, if Wi-Fi is off (0) and only Etnernet is on, then after restart the web server starts at ip 0.0.0.0, and can't be reached at eth ip address:

00:00:15.315 RSL: RESULT = {"Webserver":"Active for Admin on tasmota-C0CAD4-2772 with IP address 0.0.0.0"}

how can this be fixed?

**Comment by plewka** [comment, marked as answer]
For me it works, but I noticed the IP to be wrong/unset in the discovery message.
I would expect there is some kind of racing condition since the device fully works fine 
via Ethernet with WLAN disabled. 
I didn't get deeper into that issue up to now - didn't care too much... 
I even recognized the name resolve didn't work for the MQTT broker server. I have to 
have a closer look if this happens with WLAN, too. Could be an issue with that network
and not Tasmota's fault. Since Ping to IP worked - I used that one for first.
Are you sure the Webserver really doesn't work on an (other) IP assigned by your router?

**Comment by barbudor** [comment]
Dear all
Can you please elaborate your use case for Ethernet ?
- Ethernet only or Simultaneous Ethernet+Wifi
- What would be the network configuration ? Both interface in same subnet ? Or separated subnet ?
- In the 2nd case, please alaborate why you would like to have the same device to belong to 2 different networks ?

**Reply by sfromis** [reply]
Where I see a benefit in having Ethernet is using this instead of Wifi, in case of locations with bad wifi availability/stability, or if I want to mitigate potential interference with other 2.4 GHz activity, including BLE and Zigbee, very nearby or even the same device. Maybe with PoE, to both have better performance and power available. Two network interfaces on the same router and subnet.

For these setups, I'd disable Wifi normally, but with a bot to flip it on after restart, in case of missing Ethernet connection. More convenient than having to connect via serial. Unplug RJ45 and power cycle should make it available. In shorter words, Wifi as fallback.

It could also be if/where security considerations weight against using Wifi, but that's not my situation. And I don't believe in [EHS](https://en.wikipedia.org/wiki/Electromagnetic_hypersensitivity).
